.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: php
+
+php:
+  - '5.4'
+  - '5.5'
+  - '5.6'
+  - '7.0'
+
+before_script:
+  - git clone https://github.com/glpi-project/glpi -b 9.1 ../glpi && cd ../glpi
+  - mv ../formcreator plugins/formcreator
+  - composer install --no-dev
+  - php tools/cliinstall.php --db=glpi-test --user=travis --tests
+
+script:
+  - cd plugins/formcreator
+  - php tools/cliinstall.php --tests
```

Comment:
Why to run cliinstall another time here? If it has been runned before script, it should be useless here.